config/dynamic.c

```diff
@@ -16,11 +16,527 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <unistd.h>
+#include <arpa/inet.h>
+#include <sys/un.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <sys/select.h>
+#include <lualib.h>
+#include <lauxlib.h>
+
+#include <rte_log.h>
+#include <rte_lcore.h>
+#include <rte_cycles.h>
+#include <rte_malloc.h>
+
+#include "gatekeeper_net.h"
+#include "gatekeeper_main.h"
 #include "gatekeeper_config.h"
+#include "gatekeeper_launch.h"
+
+/* TODO Get the install-path via Makefile. */
+#define LUA_DY_BASE_DIR     "./lua"
+#define DYNAMIC_CONFIG_FILE "dy.lua"
+
+const char *CLIENT_PROC_ERROR =
+	"function reply() return -1, \"Dynamic Configuration cannot process the request. end\"\n";
+
+const uint16_t MSG_MAX_LEN = (uint16_t)~0U;
```

Comment:
Maybe it's because of the strict compilation check, without ``uint16_t``, it pops an error message ``error: large integer implicitly truncated to unsigned type [-Werror=overflow]``.